### PR TITLE
add Nandani-Paliwal profile

### LINKED ISSUE #760: Nandani
github readme

### PR DESCRIPTION
**Pull request by Nandani-Paliwal**
resolve #760

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1420,5 +1420,11 @@
     "githubUsername": "cankush625",
     "imageUrl": "https://user-images.githubusercontent.com/41515472/135764171-9fbac687-0017-449f-815d-527dc6b5f44f.jpg",
     "issueNumber": 754
+  },
+ {
```
Comment (deephunt3r) [inline data.json:1424]:
```suggestion
  {
```

Comment (deephunt3r) [reply data.json:1424]: A small change!

**Comment by deephunt3r** [comment]
Hey @Nandani-Paliwal ,I just approved the changes! The maintainers will review and they will merge it soon! 😄

File: data.json [hunk 2]
```diff
@@ -1420,5 +1420,11 @@
     "githubUsername": "cankush625",
     "imageUrl": "https://user-images.githubusercontent.com/41515472/135764171-9fbac687-0017-449f-815d-527dc6b5f44f.jpg",
     "issueNumber": 754
+  },
+  {
+    "name": "Nandani Paliwal",
+    "githubUsername": "Nandani-Paliwal",
+    "imageUrl":"https://user-images.githubusercontent.com/83964826/136408997-e2fcefa5-2c78-4484-9ac3-574bd72da253.PNG",
```
Comment (Panquesito7) [inline data.json:1427]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/83964826/136408997-e2fcefa5-2c78-4484-9ac3-574bd72da253.PNG",
```

File: data.json [hunk 3]
```diff
@@ -1420,5 +1420,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/41515472/135764171-9fbac687-0017-449f-815d-527dc6b5f44f.jpg",
     "issueNumber": 754
-  }
+  },
+  {
+    "name": "Nandani Paliwal",
+    "githubUsername": "Nandani-Paliwal",
+    "imageUrl": "https://user-images.githubusercontent.com/83964826/136408997-e2fcefa5-2c78-4484-9ac3-574bd72da253.PNG",
+    "issueNumber": 760
+  },
```
Comment (schmelto) [inline data.json:1429]:
```suggestion
  }
```

**Pull request closed without merging**